pkg/cloudprovider/providers/azure/azure_vmss.go

```diff
@@ -139,7 +139,7 @@ func (ss *scaleSet) getVmssVMByInstanceID(resourceGroup, scaleSetName, instanceI
 	}
 
 	if cachedVM == nil {
-		glog.Errorf("cound't find vmss virtual machine by scaleSetName (%q) and instanceID (%q)", scaleSetName, instanceID)
+		glog.Errorf("could't find vmss virtual machine by scaleSetName (%q) and instanceID (%q)", scaleSetName, instanceID)
```

Comment:
couldn't

Reply:
can we change the %q to %s? the scaleSetName and instanceID are all string type, and no need to print within '' IMO.